src/index.html

```diff
@@ -7,8 +7,28 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header>      
+      <nav class="nav-bar">
+        <div class="logo">
+          <a href="#home"><img src="./images/logo.png" alt="MOYO logo"></a>
```

Comment:
```suggestion
          <a href="#home">
            <img src="./images/logo.png" alt="MOYO logo">
          </a>
```
It is better to provide newline for each tag to make your code cleaner